addons/product/views/product_views.xml

```diff
@@ -332,7 +332,7 @@
             <field name="name">Product Variants</field>
             <field name="type">ir.actions.act_window</field>
             <field name="res_model">product.product</field>
-            <field name="context">{'search_default_product_tmpl_id': [active_id], 'default_product_tmpl_id': active_id}</field>
+            <field name="context">{'search_default_product_tmpl_id': [active_id], 'default_product_tmpl_id': active_id, 'create': False'}</field>
```

Comment:
```xml
<field name="context">{
'search_default_product_tmpl_id': [active_id],
'default_product_tmpl_id': active_id, 
'create': False
}</field>
```
There is a **` ' `** too much after False in your context dict ;).

Reply:
@Feyensv Done